src/policyAgent/__tests__/index.test.ts

```diff
@@ -1,23 +1,25 @@
 import { expect } from 'chai'
 import { describe, it } from 'mocha'
 
+import PinoLogger from '../../utils/logger.js'
 import PolicyAgent from '../index.js'
 import { mockEnv, withEvaluateResponse, withGetPoliciesResponse, withGetPolicyResponse } from './fixtures/mock.js'
 
 const exampleId = 'example.rego'
 const examplePackageId = 'example'
+const logger = new PinoLogger('silent')
 
 describe('policy agent', function () {
   describe('ctor', function () {
     it('should construct a PolicyAgent if origin is valid', function () {
-      const policyAgent = new PolicyAgent(mockEnv('https://example.com'))
+      const policyAgent = new PolicyAgent(mockEnv('https://example.com'), logger)
```

Comment:
why do we need the extra arg on `PolicyAgent` when you haven't modified the class constructor as far as I can tell?

Reply:
since we are a little easier on `npm t` when it comes to types checking so it has been going on for a little while and can be seen in actions tab (just added a screenshot here)
<img width="1214" alt="Screenshot 2024-09-19 at 15 58 03" src="https://github.com/user-attachments/assets/042cc6fa-03c6-4731-b451-dcc254259c38">

Happy to revert

Reply:
no if it wants the extra arg please keep it